swagger2locustio/__main__.py

```diff
@@ -101,6 +104,107 @@ def main():
     except ValueError as error:
         logging.error(error)
 
+    log_diff(main_start, log_result_named(args.results_path), args.results_path)
+    # log_result(args.results_path)
+
+
+def log_diff(start, end, results_path):
+    """Function: log difference"""
+
+    for key, items in start.items():
+        start_key = set(items)
+        end_key = set(end[key])
+        result = {
+            "created": [],
+            "unchanged": [],
+            "updated": [],
+            "deleted": [],
+        }
+
+        # CREATED / DELETED
+        # cutting of multiline to avoid issues with code that was changed
+        start_key_names = {x[: x.find("\n")] if x.find("\n") != -1 else x for x in start_key}
+        end_key_names = {x[: x.find("\n")] if x.find("\n") != -1 else x for x in end_key}
+        result["created"] = list(end_key_names - start_key_names)
+        result["deleted"] = list(start_key_names - end_key_names)
+
+        # UNCHANGED / UPDATED
+        for each_start in start_key:
+            for each_end in end_key:
+                if each_start == each_end:
+                    # folder entries do not include \n, so last letter is being cut off
+                    result["unchanged"].append(
+                        each_start[: each_start.find("\n")] if each_start.find("\n") != -1 else each_start
+                    )
+                elif each_start[: each_start.find("\n")] == each_end[: each_end.find("\n")]:
+                    result["updated"].append(each_start[: each_start.find("\n")])
+                # else is not used as we compare two lists which includes a lot of false entries
+
+        for result_key in result:
+            result[result_key].sort()
+            result_len = len(result[result_key])
+            if result_len != 0 and (key != "folders" or result_key != "updated"):
+                logging.info("%s %s: %d", key.upper(), result_key, result_len)
+                if result_key != "unchanged":
+                    logging.debug("%s %s items:", key.upper(), result_key)
+                    for each in result[result_key]:
+                        logging.debug("    %s", each)
+
+    logging.info("NOTE: Please make sure to fill in the constant files. Feel free to use helper functions to do it")
+    logging.info("NOTE: We also advise to check authorization settings")
+    logging.debug("NOTE: All the paths mentioned use %s as root directory", results_path)
+    # logging.info("==== NOTE: ====")
+    # logging.info("- Please make sure to fill in the constant files. Feel free to use helper functions to do it")
+    # logging.info("- We also advise to check authorization settings")
+    # logging.debug("- All the paths mentioned use %s as root directory", results_path)
+
+
+def log_result_named(results_path):
```

Comment:
I do not see any major reason for it to be exactly here or not, so it can be easily migrated somewhere else. I suppose that discussing it during meeting is the simplest solution